plugins/lookup/onepassword_raw.py

```diff
@@ -96,8 +102,10 @@ def run(self, terms, variables=None, **kwargs):
         secret_key = self.get_option("secret_key")
         master_password = self.get_option("master_password")
         service_account_token = self.get_option("service_account_token")
+        connect_host = self.get_option("connect_host")
+        connect_token = self.get_option("connect_token")
```

Comment:
Do a sanity check here to ensure both of these are provided.
```suggestion
        connect_token = self.get_option("connect_token")
        
        if (connect_host or connect_token) and None in (connect_host, connect_token):
            raise AnsibleOptionsError("connect_host and connect_token are required together")
```